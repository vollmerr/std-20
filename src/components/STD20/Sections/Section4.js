import React from 'react';
import { FieldInput } from 'state-template';

import schema from '../schema';

const Section4 = () => (
  <div className={'row'} role={'group'} aria-labelledby={'section4__header'}>
    <div id={'section4__header'} className={'col-md-12'}>4. Agency Information</div>

    <FieldInput {...schema.agencyDepartment} className={'col-md-4'} />
    <FieldInput {...schema.agencyContact} className={'col-md-4'} />
    <FieldInput {...schema.agencyPhone} className={'col-md-2'} />
    <FieldInput {...schema.agencyFax} className={'col-md-2'} />

    <FieldInput {...schema.agencyDivision} className={'col-md-4'} />
    <FieldInput {...schema.gsaCode} className={'col-md-4'} />
    <FieldInput {...schema.agencyEmail} className={'col-md-4'} />

    <FieldInput {...schema.agencyServiceAddress} className={'col-md-4'} />
    <FieldInput {...schema.agencyRequestedAddress} className={'col-md-4'} />
    <FieldInput {...schema.agencyBillingAddress} className={'col-md-4'} />
  </div>
);

export default Section4;
