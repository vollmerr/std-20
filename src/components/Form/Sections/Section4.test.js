import React from 'react';
import { shallow } from 'enzyme';

import Section4 from './Section4';

const props = {};

let wrapper;
describe('Section4', () => {
  beforeEach(() => {
    wrapper = shallow(<Section4 {...props} />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
