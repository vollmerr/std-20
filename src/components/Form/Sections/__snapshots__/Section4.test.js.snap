// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section4 should render correctly 1`] = `
<div
  aria-labelledby="section4__header"
  className="row"
  role="group"
>
  <div
    className="col-md-12"
    id="section4__header"
  >
    4. Agency Information
  </div>
  <Component
    className="col-md-4"
    label="Department"
    name="agencyDepartment"
  />
  <Component
    className="col-md-4"
    label="Contact Name"
    name="agencyContact"
  />
  <Component
    className="col-md-2"
    label="Phone Number"
    name="agencyPhone"
    normalize={[Function]}
  />
  <Component
    className="col-md-2"
    label="Fax Number"
    name="agencyFax"
    normalize={[Function]}
  />
  <Component
    className="col-md-4"
    label="Division (Unit, etc.)"
    name="agencyDivision"
  />
  <Component
    className="col-md-4"
    label="General Services Agency Code"
    name="gsaCode"
  />
  <Component
    className="col-md-4"
    label="Email Address"
    name="agencyEmail"
  />
  <Component
    className="col-md-4"
    label="Present Service Address"
    name="agencyServiceAddress"
    tag="textarea"
  />
  <Component
    className="col-md-4"
    label="Requested Service Address"
    name="agencyRequestedAddress"
    tag="textarea"
  />
  <Component
    className="col-md-4"
    label="Billing Address"
    name="agencyBillingAddress"
    tag="textarea"
  />
</div>
`;
